requirements.txt

```diff
@@ -12,4 +12,4 @@ cosl ==0.0.15
 # juju 3.1.2.0 depends on pyyaml<=6.0 and >=5.1.2
 PyYAML ==6.0.*
 pyOpenSSL==24.2.1
-github_runner_manager @ git+https://github.com/canonical/github-runner-manager.git@6f276fd0fc95ed3550005a0b394225d1b506f7dd
+github_runner_manager @ git+https://github.com/canonical/github-runner-manager.git@cfb8f74b93d6f29acce36cfc87ccab6d7c19a344
```

Comment:
I guess we need to wait for the manager PR to be done?

Reply:
the one is merged already : https://github.com/canonical/github-runner-manager/pull/29